### PR TITLE
Feature (zeb-996): test runs tuning

### PR DESCRIPTION
**Pull request by danymill**
image
image

### CODE REVIEW

File: client/app/containers/tests-runs/tests-runs-search/modal/search-modal.controller.js [hunk 1]
```diff
@@ -3,22 +3,23 @@
...
-    windowWidthService,
     DEFAULT_SC,
     UtilService,
     ) {
     'ngInject';
 
     const STATUSES = ['PASSED', 'FAILED', 'SKIPPED', 'ABORTED', 'IN_PROGRESS', 'QUEUED', 'UNKNOWN'];
     const vm = {
-        isMobile: windowWidthService.isMobile,
+        get isMobile() { return $mdMedia('xs'); },
```
Comment (DmitryMishin) [inline client/app/containers/tests-runs/tests-runs-search/modal/search-modal.controller.js:22]: you don't use that in template and controller

File: client/app/shared/test-run-card/test-run-card.controller.js [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+    authService,
+    DownloadService,
+    messageService,
+    SlackService,
+    TestRunService,
+    testsRunsService,
+    toolsService,
+    UserService,
+    UtilService,
+    ) {
```
Comment (DmitryMishin) [inline client/app/shared/test-run-card/test-run-card.controller.js:20]: incorrect spaces

File: client/app/shared/test-run-card/test-run-card.controller.js [hunk 3]
```diff
@@ -0,0 +1,436 @@
...
+        }
+
+        function checkFilePresence() {
+            if (!vm.testRun.appVersionValid) {
+                vm.testRun.appVersionLoading = true;
+                DownloadService.check(vm.testRun.config.appVersion).then(function (rs) {
+                    if (rs.success) {
+                        vm.testRun.appVersionValid = rs.data;
+                    } else {
+                        //messageService.error(rs.message);
```
Comment (DmitryMishin) [inline client/app/shared/test-run-card/test-run-card.controller.js:414]: do you need that?

**Review by dzmitrymishyn — APPROVED** [review]

File: client/app/shared/test-run-card/test-run-card.controller.js (hunk 2 above)
Comment (danymill) [reply client/app/shared/test-run-card/test-run-card.controller.js]: fixed

File: client/app/shared/test-run-card/test-run-card.controller.js (hunk 3 above)
Comment (danymill) [reply client/app/shared/test-run-card/test-run-card.controller.js]: fixed

**Pull request merged**